torch/_dynamo/output_graph.py

```diff
@@ -529,9 +529,16 @@ def call_user_compiler(self, gm: fx.GraphModule) -> CompiledFn:
             )
             _step_logger()(logging.INFO, f"calling compiler function {name}")
             compiler_fn = self.compiler_fn
+            # WrapperBackend needs real inputs, for now, to verify correctness
             if config.verify_correctness:
                 compiler_fn = WrapperBackend(compiler_fn, self.example_inputs())
-            compiled_fn = compiler_fn(gm, self.fake_example_inputs())
+
+            # We invoke the backend with real inputs if and only if
+            # we are in doing accuracy evaluation in the minifier
+            if config.repro_after is not None and config.repro_level == 4:
+                compiled_fn = compiler_fn(gm, self.example_inputs())
+            else:
+                compiled_fn = compiler_fn(gm, self.fake_example_inputs())
```

Comment:
I thought the PoR was we were going to integrate WrapperBackend to become a runtime concept